.circleci/config.yml

```diff
@@ -64,7 +64,7 @@ variables:
 jobs:
   build:
     docker:
-      - image: circleci/python:3.5-jessie
+      - image: circleci/python:3.5.7-stretch
```

Comment:
I'm tempted to only support 3.6 or 3.7 in general -- wdyt?

Reply:
I think it'd be fine to just to support 3.7.